docs/README.md

```diff
@@ -0,0 +1,20 @@
+# Scraper Development Guide
+
+## Goal
+
+This library has the goals of
+
+* making recipe information **accessible**,
+* ensuring the author is **attributed** correctly,
+* representing the recipes **accurately** and **authentically**
+
+Sometimes it is simple and straightforward to achieve all these goals, and sometimes it is more difficult (which is why this library exists). Where some interpretation or creativity is required to scrape a recipe, we should always keep those goals in mind. Occasionally, that might mean that we can't support a particular website.
```

Comment:
+1 @strangetom - this is a nice way of explaining the goals.  I agree with placing accuracy and authenticity in the same line item, too - I was thinking about exactly that during a commute recently and figured that those two reduce to more-or-less the same idea.

Reply:
This was very much inspired by comment you made on a recent PR :)

Reply:
Thanks!  I hope they're useful guidelines - I don't want to have too much influence, though! :smile:

@hhursev @bfcarpio @brett as some of the more frequent committers here, do these more-or-less match the aims you have in mind when writing scrapers?  And/or any suggestions for adjustments?